src/Operators/integrals.jl

```diff
@@ -5,37 +5,50 @@ import ClimaComms
 """
     column_integral_definite!(ϕ_top, ᶜ∂ϕ∂z, [ϕ_bot])
 
-Sets `ϕ_top```{}= \\int_{z_{bot}}^{z_{top}}\\,```ᶜ∂ϕ∂z```(z)\\,dz +{}```ϕ_bot`,
-where ``z_{bot}`` and ``z_{top}`` are the values of `z` at the bottom and top of
-the domain, respectively. The input `ᶜ∂ϕ∂z` should be a cell-center `Field` or
-`AbstractBroadcasted`, and the output `ϕ_top` should be a horizontal `Field`.
-The default value of `ϕ_bot` is 0.
+Sets `ϕ_top```{}= \\frac{1}{ΔA(z_{bot})}\\int_{z_{bot}}^{z_{top}}\\,
+```ᶜ∂ϕ∂z```(z)\\,ΔA(z)\\,dz +{}```ϕ_bot`, where ``z_{bot}`` and ``z_{top}`` are
+the values of `z` at the bottom and top of the domain, and where `ΔA(z)` is the
+area differential `J(z)/Δz`, with `J(z)` denoting the metric Jacobian. The input
```

Comment:
```suggestion
area differential `J(z)/Δz(z)`, with `J(z)` denoting the metric Jacobian. The input
```
both are a function of `z`, correct?

Reply:
Yes, they are both functions of `z`. But there were too many `z`s in this docstring so I've since removed them to improve readability.